src/brain/index_tuner.cpp

```diff
@@ -122,7 +122,7 @@ void IndexTuner::BuildIndex(storage::DataTable* table,
       key->SetFromTuple(tuple_ptr.get(), indexed_columns, index->GetPool());
 
       // Insert in specific index
-      index->InsertEntry(key.get(), location);
+      // index->InsertEntry(key.get(), location);
```

Comment:
it's not unused code. it's temporarily commented out because of the indirection layer refactor. will be restored in next PR.